api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -105,6 +106,17 @@ public void setCheckIssuer(boolean checkIssuer) {
         this.checkIssuer = checkIssuer;
     }
 
+    @Description("Enable or disable audience checking. If audience checking is enabled the `clientId` also has to be configured. " +
```

Comment:
Ah, so I was confused because I thought this must be a _Kafka_ client id. It makes more sense now. 

```suggestion
    @Description("Enable or disable audience checking. If audience checking is enabled the OAuth Client ID also has to be configured via the `clientId` property. " +
```